README.md

```diff
@@ -143,24 +143,35 @@ Video/voice is also super simple! In this example, peer1 sends video to peer2.
 var Peer = require('simple-peer')
 
 // get video/voice stream
-navigator.getUserMedia({ video: true, audio: true }, gotMedia, function () {})
+navigator.getUserMedia({ video: true, audio: true }, gotMedia, () => {})
 
 function gotMedia (stream) {
   var peer1 = new Peer({ initiator: true, stream: stream })
   var peer2 = new Peer()
 
-  peer1.on('signal', function (data) {
+  peer1.on('signal', data => {
     peer2.signal(data)
   })
 
-  peer2.on('signal', function (data) {
+  peer2.on('signal', data => {
     peer1.signal(data)
   })
 
-  peer2.on('stream', function (stream) {
+  peer2.on('stream', stream => {
     // got remote video stream, now let's show it in a video tag
     var video = document.querySelector('video')
-    video.src = window.URL.createObjectURL(stream)
+    
+    // This detection is optional and is just here to be retrocompatible with some olders browsers:
+    // May can be removed
+    if (video.srcObject) {
```

Comment:
This check actually isn't correct as `srcObject` is initialized to `null`. It should be:
```javascript
if ('srcObject' in video)
```